goon/code/datums/browserOutput.dm

```diff
@@ -221,7 +221,7 @@ For the main html chat area
 
 	// Either an atom or somebody fucked up and is gonna get a runtime, which I'm fine with.
 	var/atom/A = obj
-	var/key = "[istype(A.icon, /icon) ? "\ref[A.icon]" : A.icon]:[A.icon_state]"
```

Comment:
Rather than completely removing this check, I recommend replacing it with `"[A.icon]" || "\ref[A.icon]"`
That makes it keep the INTENDED current functionality, except it actually works